content/en/serverless/azure_app_services.md

```diff
@@ -11,186 +11,126 @@ further_reading:
-                  "eb7c615e5fca779871203b7de9209b6c",
-                  source: "<SOURCE_NAME>",
-                  service: "<SERVICE_NAME>",
-                  tags: new string[] { "<TAG_1>:<VALUE_1>", "<TAG_2>:<VALUE_2>" },
-                  configuration: config
-              )
-              .Enrich.FromLogContext()
-              .CreateLogger();
-{{< /code-block >}}
-
-New Logs are now directly sent to Datadog.
-
-Alternatively, since 0.2.0, configure the Datadog sink by using an `appsettings.json` file with the Serilog.Settings.Configuration NuGet package.
-
-In the `Serilog.WriteTo()` array, add an entry for DatadogLogs, for example:
-
-```json
-"Serilog": {
-  "Using": [ "Serilog.Sinks.Console", "Serilog.Sinks.Datadog.Logs" ],
-  "MinimumLevel": "Debug",
-  "WriteTo": [
-    { "Name": "Console" },
-    {
-      "Name": "DatadogLogs",
-      "Args": {
-        "apiKey": "<API_KEY>",
-        "source": "<SOURCE_NAME>",
-        "host": "<HOST_NAME>",
-        "tags": ["<TAG_1>:<VALUE_1>", "<TAG_2>:<VALUE_2>"],
-      }
-    }
-  ],
-  "Enrich": [ "FromLogContext", "WithMachineName", "WithThreadId" ],
-  "Properties": {
-    "Application": "Sample"
-  }
-}
-```
-
-## Custom Metrics with DogStatsD
-
-Starting with version `0.3.14-prerelease`, the App Services extension includes an instance of [DogStatsD][6] (Datadog's metrics aggregation service). This enables you to submit custom metrics, service checks, and events directly to Datadog from Azure Web Apps with the extension.
-
-Writing custom metrics and checks in your web app is similar to the process for doing so with an application on a host running the Datadog Agent. However, there is no need to configure ports, which is taken care of by the extension automatically. To submit custom metrics to Datadog from Azure App Services using the extension:
-
-1. Add the [DogStatsD Nuget package][13] to your Visual Studio project.
-2. [Initialize DogStatdD and write custom metrics][14] in your application.
+1. Add the [DogStatsD NuGet package][14] to your Visual Studio project.
+2. Initialize DogStatdD and write custom metrics in your application.
 3. Deploy your code to a supported Azure .NET web app.
 4. Install the Datadog App Service extension.
 
-Learn more about [custom metrics][15].
-
-## Troubleshooting
-
-### 5XX errors
+**Note**: Unlike the [standard DogStatsD config process][15], there is no need to set ports or a server name when initializing the DogStatsD configuration. There are ambient environment variables in Azure App Service that determine how the metrics are sent (requires v6.0.0+ of the DogStatsD client).
 
-If your app begins throwing 5XX errors immediately after installation, first try to reinstall with the application in a full stop. To do this:
+To send metrics use this code:
 
-1. Stop the application.
-2. Remove the Datadog extension.
-3. Reinstall the Datadog extension.
-4. Restart your application.
+```csharp
+using (var statsService = new DogStatsdService())
```

Comment:
@ruthnaebeck we have an updated example code block we think will make this more clear + help prevent tickets from a transient error we have seen in some cases:

try
{
    // Configure your DogStatsd client and configure any tags
    DogStatsd.Configure(new StatsdConfig() { ConstantTags = new[] { "app:sample.mvc.aspnetcore" } });
}
catch (Exception ex)
{
    // An exception will be thrown by the Configure call if the necessary environment variables are not present.
    // These environment variables are present in Azure App Services
    // But will need to be set in order to test your custom metrics: DD_API_KEY:{api_key}, DD_AGENT_HOST:localhost 
    // Ignore or log the exception as it suits you
    Console.WriteLine(ex);
}
// Send a metric
DogStatsd.Increment("sample.startup");